modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptRxjsClientCodegen.java

```diff
@@ -308,24 +311,29 @@ private void addConditionalImportInformation(Map<String, Object> operations) {
     }
 
     private void addExtraReservedWords() {
-        this.reservedWords.add("BASE_PATH");
-        this.reservedWords.add("BaseAPI");
-        this.reservedWords.add("RequiredError");
-        this.reservedWords.add("COLLECTION_FORMATS");
-        this.reservedWords.add("ConfigurationParameters");
-        this.reservedWords.add("Configuration");
-        this.reservedWords.add("HttpMethod");
-        this.reservedWords.add("HttpHeaders");
-        this.reservedWords.add("HttpQuery");
-        this.reservedWords.add("HttpBody");
-        this.reservedWords.add("ModelPropertyNaming");
-        this.reservedWords.add("RequestArgs");
-        this.reservedWords.add("RequestOpts");
-        this.reservedWords.add("ResponseArgs");
-        this.reservedWords.add("exists");
-        this.reservedWords.add("Middleware");
-        this.reservedWords.add("AjaxRequest");
-        this.reservedWords.add("AjaxResponse");
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseSensitive(
```

Comment:
case sensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)